jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -783,9 +784,7 @@ private String replaceReservedChars(String str) {
   }
 
   @Override
-  public InterpreterResult interpret(String originalCmd, InterpreterContext contextInterpreter) {
```

Comment:
Hi @mebelousov the change proposed in these lines removes the capability described in [ZEPPELIN-3377](https://issues.apache.org/jira/browse/ZEPPELIN-3377) (Passing Z variables to JDBC interpreter) and the corresponding [PR-2903](https://github.com/apache/zeppelin/pull/2903). 

Is it necessary to remove ZEPPELIN-3377 capability to implement this PR?